pkg/vpa/vpa.go

```diff
@@ -34,6 +34,7 @@ type Reconciler struct {
 	KubeClient        *kube.ClientInstance
 	VPAClient         *kube.VPAClientInstance
 	OnByDefault       bool
+	DryRun            bool
```

Comment:
Another great change. Dryrun functionality came before this was actually a  struct, so I think it just got forgotten.